ppr-ui/src/utils/auth-helper.ts

```diff
@@ -181,8 +181,7 @@ export async function getSbcFromAuth (): Promise<boolean> {
         if (!data) {
           return false
         }
-        const branchName = data?.branchName
-        if (branchName?.includes('Service BC')) {
+        if (data.orgType === AccountTypes.SBC_STAFF) {
```

Comment:
Just to verify, this `orgType` has been set in all deployed environments? 
ie when this change goes to PROD, SBC Staff will still have their access?

Reply:
I am assuming so... The change has already been made to the API and I believe it is in production..